pkg/cmd/pr/merge/merge.go

```diff
@@ -468,6 +473,19 @@ func blockedReason(status string) string {
 	}
 }
 
+func isBlocked(status string, admin bool) bool {
+	switch status {
+	case "BLOCKED":
+		return !admin
+	case "BEHIND":
```

Comment:
Does BEHIND allow the admin override? I've never tested this in the web UI.

Reply:
Oh that was just my assumption to what you wrote in https://github.com/cli/cli/issues/1198#issuecomment-886745251.
Probably wrong then.

Reply:
According to https://stackoverflow.com/questions/34463895/git-this-branch-is-behind-after-pull-request `BEHIND` seems to be rather informational only

Reply:
Not too sure how to force the `BEHIND` state yet (for a test).